dev-tools/omdb/src/bin/omdb/db.rs

```diff
@@ -2125,20 +2125,17 @@ async fn cmd_db_disk_info(
         physical_disk: String,
     }
 
-    use nexus_db_schema::schema::disk::dsl as disk_dsl;
-
     let conn = datastore.pool_connection_for_tests().await?;
 
-    let disk = disk_dsl::disk
-        .filter(disk_dsl::id.eq(args.uuid))
-        .limit(1)
-        .select(Disk::as_select())
-        .load_async(&*conn)
-        .await
-        .context("loading requested disk")?;
+    let disk_name = disk.name().to_string();
 
-    let Some(disk) = disk.into_iter().next() else {
-        bail!("no disk: {} found", args.uuid);
+    let volume_id = disk.volume_id().to_string();
+
+    let disk_state = disk.runtime().disk_state.to_string();
+
+    let import_address = match disk.pantry_address() {
```

Comment:
I came here to complain about this, but when I actually see what you did, it's right and
what I had before was not, thanks for fixing it.